ding/world_model/dreamer.py

```diff
@@ -172,9 +192,32 @@ def train(self, env_buffer, envstep, train_iter, batch_size, batch_length):
         data = {k: torch.stack(data[k], dim=1) for k in data}  # -> {dict_key: Tensor([B, T, any_dims])}
 
         data['discount'] = data.get('discount', 1.0 - data['done'].float())
-        data['discount'] *= 0.997
+        # data['discount'] *= 0.997
```

Comment:
why comment this line

Reply:
This operation is no use. This operation only changes the model's predicted end-of-environment signal from 1 to 0.997.